duckduckgo/iOS
# [DuckPlayer] 23. Ship Review Fixes

Task/Issue URL: https://app.asana.com/0/1204099484721401/1208094354951735/f

**Description**:
Fixes issues found in DuckPlayer ship review
- Opens videos in Youtube app if present and user selects "Open in Youtube"
- Updates bookmarks images
- Updates history



**Steps to test this PR**:
1. Make yourself internal user
2. Add this custom config https://www.jsonblob.com/api/1276215786342309888 
3. Add a bookmark and a favorite while watching a video in DuckPlayer
4. Confirm custom icons are used

----

1. Disable Duck Player
2. Go to use-devtesting18.duckduckgo.com and search for a video
3. Tap "Watch in Youtube"
4. In the simulator of if Youtube app is not installed, video should load in DDG Browser
5. If Youtube app is installed, video should load in YT app

---

1. Set DuckPlayer to 'always ask'
2. Go to Youtube
3. Search for a video
4. Tap "Watch in Duck Player" (Dont enable remember choice)
5. Go back
6. Confirm you're taken to the video page and not results view.
